plugin.video.dailymotion_com/default.py

```diff
@@ -590,13 +632,14 @@ def addUserDir(name, url, mode, iconimage, desc):
                 'fanart': _fanart})
     liz.setInfo(type="Video", infoLabels={"Title": name, "Plot": desc})
     if dmUser == "":
-        playListInfos = "###MODE###=ADD###USER###="+name+"###THUMB###="+iconimage+"###END###"
-        liz.addContextMenuItems([(translation(30028), 'XBMC.RunPlugin(plugin://plugin.video.dailymotion_com/?mode=favourites&url='+urllib.parse.quote_plus(playListInfos)+')',)])
+        playListInfos = "###MODE###=ADD###USER###=" + name + "###THUMB###=" + iconimage + "###END###"
+        liz.addContextMenuItems([(translation(30028), 'XBMC.RunPlugin(plugin://plugin.video.dailymotion_com/?mode=favourites&url=' + urllib.parse.quote_plus(playListInfos) + ')',)])
```

Comment:
Since you are refactoring code, try to change `XBMC.RunPlugin` to just `RunPlugin`